app/src/main/java/org/projectbuendia/client/sync/SyncAdapter.java

```diff
@@ -103,14 +69,23 @@
      * Select a phase by setting a boolean value of true for the appropriate key.
      */
     public enum SyncPhase {
-        SYNC_USERS,
-        SYNC_LOCATIONS,
-        SYNC_CHART_ITEMS,
-        SYNC_CONCEPTS,
-        SYNC_PATIENTS,
-        SYNC_OBSERVATIONS,
-        SYNC_ORDERS,
-        SYNC_FORMS
+        SYNC_USERS(R.string.syncing_users, new UsersSyncPhaseController()),
```

Comment:
This is great!  So nice to see each phase in its own class.

Why not go all the way and make them all SyncPhases?  SyncPhases and SyncPhaseControllers don't need to be two separate things; couldn't they be a single interface? As in:

    interface SyncPhase {
        String getName();
        String getMessage();
        void sync();
    }